zetaclient/orchestrator/bootstrap.go

```diff
@@ -377,6 +379,7 @@ func syncObserverMap(
 				*params,
 				client,
 				tss,
+				time.Duration(cfg.RPCAlertLatency),
```

Comment:
same here

Reply:
same, I also unified a bit more in commit https://github.com/zeta-chain/node/pull/2751/commits/05139aa9dd5c14cde3fd81fe4ec3f1fb1202dbe6